lib/core/presentation/widgets/bottom_bar/bottom_bar_widget.dart

```diff
@@ -0,0 +1,182 @@
+        ),
+      ),
+      child: Column(
+        children: [
+          SizedBox(height: 15.h),
+          Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: tabs.map((tabData) {
+              return _buildTabItem(context, tabData);
+            }).toList(),
+          ),
+          SizedBox(height: 18.h),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildTabItem(BuildContext context, TabData tabData) {
+    final isSelected = _selectedTab == tabData.tab;
+    return _buildItem(context, tabData, isSelected);
+  }
+
+  Widget _buildItem(BuildContext context, TabData tabData, bool isSelected) {
+    final icon = _buildIcon(context, tabData, isSelected);
+    return Expanded(
+      child: GestureDetector(
+        behavior: HitTestBehavior.opaque,
+        onTap: () {
+          _handleTabTap(context, tabData);
+        },
+        child: Container(
+          color: Colors.transparent,
+          child: Stack(
+            alignment: Alignment.center,
+            children: [
+              _buildAnimatedContainer(isSelected),
+              icon,
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildIcon(BuildContext context, TabData tabData, bool isSelected) {
+    final icon = tabData.icon;
+    final selectedIcon = tabData.selectedIcon;
+    // Handle display user avatar for tab profile
+    if (tabData.tab == AppTab.profile) {
+      return BlocBuilder<AuthBloc, AuthState>(
+        builder: (context, authState) {
+          if (authState is AuthStateAuthenticated) {
+            return Center(
+              child: LemonCircleAvatar(
+                size: 24.w,
+                url: AvatarUtils.getProfileAvatar(
+                  userAvatar: authState.authSession.userAvatar,
+                  userId: authState.authSession.userId,
+                ),
+              ),
+            );
+          } else if (authState is AuthStateProcessing) {
+            return Loading.defaultLoading(context);
+          } else {
+            final colorScheme = Theme.of(context).colorScheme;
+            return Icon(Icons.person, color: colorScheme.onPrimary, size: 24);
+          }
+        },
+      );
+    } else {
+      return isSelected ? selectedIcon : icon;
+    }
+  }
+
+  void _handleTabTap(BuildContext context, TabData tabData) {
+    Vibrate.feedback(FeedbackType.light);
+    setState(() {
+      _isTabChanged = true;
+      _selectedTab = tabData.tab;
+    });
+    _animationController.reset();
+    _animationController.forward();
+    if (tabData.tab == AppTab.profile) {
+      final authState = BlocProvider.of<AuthBloc>(context).state;
+      if (authState is AuthStateAuthenticated) {
+        AutoRouter.of(context)
+            .navigateNamed(tabData.route, includePrefixMatches: true);
+      } else {
+        AutoRouter.of(context).navigateNamed('/login');
+      }
+    } else {
```

Comment:
I think we should not navigate using raw string route. AutoRouter support navigate by using `context.router.push(PageRoute)`. You may try it

Reply:
Adjust to context.router.navigate(const LoginRoute()) and it's worked 👍